Snowplow/SPUtilities.m

```diff
@@ -87,13 +89,9 @@ + (NSString *) getAppleIdfa {
     NSString* idfa = nil;
 #if SNOWPLOW_TARGET_IOS || SNOWPLOW_TARGET_TV
 #ifndef SNOWPLOW_NO_IFA
-    Class ASIdentifierManagerClass = NSClassFromString(@"ASIdentifierManager");
-    if (ASIdentifierManagerClass) {
-        SEL sharedManagerSelector = NSSelectorFromString(@"sharedManager");
-        id sharedManager = ((id (*)(id, SEL))[ASIdentifierManagerClass methodForSelector:sharedManagerSelector])(ASIdentifierManagerClass, sharedManagerSelector);
-        SEL advertisingIdentifierSelector = NSSelectorFromString(@"advertisingIdentifier");
-        NSUUID *uuid = ((NSUUID* (*)(id, SEL))[sharedManager methodForSelector:advertisingIdentifierSelector])(sharedManager, advertisingIdentifierSelector);
-        idfa = [uuid UUIDString];
+    if([[ASIdentifierManager sharedManager] isAdvertisingTrackingEnabled]) {
+        NSUUID *identifier = [[ASIdentifierManager sharedManager] advertisingIdentifier];
+        idfa = [identifier UUIDString];
```

Comment:
Question here - in addition to the `if` statement which recognises `isAdvertisingTrackingEnabled`, are we also changing the code which collects the value? If so, what are we changing and why?

Reply:
Per both https://stackoverflow.com/a/2674850/2575308 and https://stackoverflow.com/a/7445419/2575308 , they're equivalent.

Reply:
Looks ok to me - @jbeemster you're probably in a better position to see if there is any potential issue with this, if you're happy I'm happy!